x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/job_service/get_jobs.ts

```diff
@@ -35,6 +41,7 @@ export const getJobsFactory = (
 ): GetJobs => async (forceRefresh = false) => {
   if (forceRefresh === true || blockRefresh === false) {
     try {
+      refreshTransformList$.next(REFRESH_TRANSFORM_LIST_STATE.LOADING);
```

Comment:
Any reason the observable is being called here directly instead of via the `useRefreshTransFormList` function?
Like ` const { isLoading, refresh } = useRefreshTransFormList(); `

Reply:
I had to use the plain observable here because this whole file is not a component or a custom hooks itself so I couldn't use hooks in this file. I'm contemplating though whether it makes sense to change these files to make them custom hooks, but not in this PR :)